src/api/app/models/project/categorized.rb

```diff
@@ -0,0 +1,7 @@
+class Project::Categorized
```

Comment:
Hm, I think this is a bit much abstraction for just one `.map`.

Reply:
Okay, I'll put it back into `Project` and we'll extract it when we have more related stuff